### ISSUE TITLE
translate pledge

### ISSUE THREAD

**Issue by whit537** [opencompany/www.opencompany.org#138]
We need to translate the modified pledge from #134 into French and Polish.

**Comment by galuszkak** [comment]
Do we know someone who know french @whit537 and could do that for us?

**Comment by whit537** [comment]
@slaivyn Based on https://github.com/gratipay/inside.gratipay.com/issues/132, would you be willing to help over here as well? :-)

**Comment by slaivyn** [comment]
@whit537 yes, of course! I'll do that soon.

**Comment by whit537** [comment]
Thanks! :dancer:

**Comment by galuszkak** [comment]
Thank @slaivyn !

**Comment by slaivyn** [comment]
I think @kyzh has already done this translation in https://github.com/opencompany/www.opencompany.org/commit/1a35b586127ca0e9e3425c1bd3530e0165c9116a?

**Comment by whit537** [comment]
@slaivyn We modified the pledge in #21. Is the French translation caught up?

**Comment by whit537** [comment]
Sorry, #21 is old. I meant #134. :)

**Comment by whit537** [comment]
And 1a35b586127ca0e9e3425c1bd3530e0165c9116a is dated after #134, and seems like it does indeed resolve the issue.

**Comment by whit537** [comment]
:)

**Comment by galuszkak** [comment]
Wait, wait for me...

**Comment by galuszkak** [comment]
Polish translation isn't updated...

**Comment by whit537** [comment]
Hah! Sorry @galuszkak. :)

**Comment by slaivyn** [comment]
Ok, fine :)
However I think the French translation could be slightly improved: some expressions used that are not exactly French idioms.
May I suggest some changes?
One particular thing: the name "the Open Company Initiative" has been translated to "l'Initiative d'Entreprise Ouverte" which is not "so French". I think it could be let untranslated like "the Open Source Initiative": "l'Open Source Initiative" in French. So we could have the same acronym.

**Comment by whit537** [comment]
@slaivyn I say go ahead and make any changes you like to the French translation. What do you need from me/us to make that happen?

**Comment by slaivyn** [comment]
Ok. I will need some help from some English native speakers: I would have questions to well understand and translate some subtleties.
Example: when you replace "pledge" by "commit" in "We who belong to the Open Company Initiative pledge to" my understanding is that you switch from a sort of moral commitment (a pledge) to something that "we who belong to the OCI" actually do on a day-to-day basis. Am I right or is my understanding to much influenced by what I want to understand? :smile:
So where should I ask this type of questions? In a new issue? It could be interesting to have those Q&A saved somewhere so a Github issue could be the right place.
Let me know :)

**Comment by whit537** [comment]
@slaivyn This is a fine place to discuss this. I haven't thought too deeply before about the difference between "commit" and "pledge." I guess to me "pledge" reminds me of things like http://givingpledge.org/, which to me feel like empty promises. Maybe commit is more mundane or doesn't carry such lofty overtones, though we do still call it a "pledge" on the about page. Using "commit" in the language of the pledge itself also made it easier to phrase the statement on the homepage and at the top of the About page and maybe elsewhere (though probably this is only a concern in English?): "companies committed to openness as a defining element in how we create value." Using "pledged" there would've definitely felt too heavy to me.

Does that help at all? :)

**Comment by slaivyn** [comment]
So it is a moral commitment?
To be sure :)

**Comment by whit537** [comment]
@slaivyn Yeah, I guess I'm basically agreeing with your distinction between moral commitment and day-to-day commitment.
